package.json

```diff
@@ -8,8 +8,8 @@
     "build-api-docs": "typedoc",
     "lint:fix": "npm run lint:eslint -- --fix && npm run lint:prettier -- --write",
     "lint:check": "npm run lint:eslint && npm run lint:prettier -- --check",
-    "lint:eslint": "eslint --config .eslintrc.js \"src/\" \"examples/\"",
-    "lint:prettier": "prettier \"{src,examples}/*.{ts,tsx,js,jsx,css}\" \"**/*.{md,mdx,yml}\"",
+    "lint:eslint": "eslint --config .eslintrc.js \"src/\" \"e2e/\" \"examples/\"",
+    "lint:prettier": "prettier \"{src,e2e,examples}/*.{ts,tsx,js,jsx,css}\" \"**/*.{md,mdx,yml}\"",
```

Comment:
I think we should really do this from the eslint and prettier configuration files, rather than via the command line options here, we should just be able to run `eslint` and it'd execute as above, same with prettier.